src/db/seeds/UpdateHawkerCentre.ts

```diff
@@ -0,0 +1,111 @@
+import axios from 'axios';
+import * as csv from 'fast-csv';
+import fs from 'fs';
+import path from 'path';
+import { GOOGLE_MAPS_API_KEY } from '../../consts';
+
+const geocode = async (address: string) => {
+  const urlForm = 'https://maps.googleapis.com/maps/api/geocode/json?';
+  const apiKey = 'key=' + GOOGLE_MAPS_API_KEY;
+  try {
+    const urlEncodedAddress = `address=${encodeURIComponent(address)}`;
+    const URL = urlForm + urlEncodedAddress + '&' + apiKey;
+    const response: any = await axios.get(URL);
+    const latLng = response.data.results[0].geometry.location;
+    return latLng;
+  } catch (err) {
+    console.log(err);
+  }
+};
+
+/*
+RegionId | Region | first 2 digits of postal code
+1 | North | 72-73 && 77-78 && 75-76
+2 | East | 46-52 && 81
+3 | West | 60-71
+4 | Central | 01-45 && 58-59
+5 | NorthEast | 53-57 && 79-80
+*/
+
+const getRegionId = (address: string) => {
+  const postalCode = address.substring(address.indexOf('S(') + 2);
+  const firstTwoDigits = parseInt(postalCode.substring(0, 2));
+  const isInNorth: Boolean = firstTwoDigits <= 78 && firstTwoDigits >= 72 && firstTwoDigits !== 74;
```

Comment:
No need type parameter, typescript can infer. Anyway, typescript types are usually written in lowercase. More info [here](https://www.typescriptlang.org/docs/handbook/basic-types.html#about-number-string-boolean-symbol-and-object)

Reply:
noted, removed thanks